slidedeck/src/slides/02-react-basics/00-react-basics.pug

```diff
@@ -246,56 +207,38 @@ section
     };
 
 section
-  h2.lower state
+  h2.lower defaultProps
```

Comment:
`props` now has a forward reference to `defaultProps`, should probably flip around so `defaultProps` backreferences `props`